enclave_wrangler/objects_api.py

```diff
@@ -353,8 +356,17 @@ def fetch_cset_and_member_objects(
     for cset in cset_versions:
         version: int = cset['properties']['codesetId']
         # Hierarchical
-        cset['expression_items']: List[Dict] = get_concept_set_version_expression_items(version, return_detail='full')
-        cset['member_items']: List[Dict] = get_concept_set_version_members(version, return_detail='full')
+        # TODO: exc handling: Will need to handle differently or do more: https://github.com/jhu-bids/TermHub/issues/451
+        try:
+            cset['expression_items']: List[Dict] = get_concept_set_version_expression_items(version, return_detail='full')
+        except EnclavePaginationLimitErr as err:
```

Comment:
@Sigfried Creating new error type and using this (maybe just temporarily) until I have a better fix in place for the pagination limit issue as it pertains to the DB refresh.